trunk/SUAVE/Components/Energy/Networks/Propulsor_Surrogate.py

```diff
@@ -0,0 +1,117 @@
+# propulsor_surrogate.py
+#
+# Created:  Mar 2017, E. Botero
+# Modified:
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+
+# package imports
+import numpy as np
+from SUAVE.Components.Propulsors.Propulsor import Propulsor
+
+from SUAVE.Core import Data
+import sklearn
+from sklearn import gaussian_process
+from sklearn import neighbors
+from sklearn import svm
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+
+# The input format for this should be Altitude, Mach, Throttle, Thrust, SFC
+
+class Propulsor_Surrogate(Propulsor):
+    def __defaults__(self): 
+        self.nacelle_diameter  = None
+        self.engine_length     = None
+        self.number_of_engines = None
+        self.tag               = 'Engine_Deck_Surrogate'
+        self.input_file        = None
+        self.sfc_surrogate     = None
+        self.thrust_surrogate  = None
+        self.thrust_angle      = 0.0
+        self.areas             = Data()
+        self.surrogate         = 'gaussian'
```

Comment:
I think self.surrogate_type would be better, self.surrogate is a little misleading.